device/globals/game.gd

```diff
@@ -15,6 +15,8 @@ var last_obj = null
 var current_action = ""
 var current_tool = null
 
+var walk_context = null
```

Comment:
Isn't this variable only used in `clicked()`?

Reply:
Yes. However, I force-pushed this change, because I got a complaint somewhere `clicked()` that the variable wasn't found! I declared it separately in both the `if` and the `else` which may have been a problem, but this is a sure-fire way of making it work.

Reply:
Yeah, but couldn't you declare it at the top of the `clicked()` function, before the if-else-statement?

Reply:
Yes. Pushed it that way now.